### PR TITLE
new release notes

### PR DESCRIPTION
**Pull request by joeyouss**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: release-notes/cloud-cost-management.md [hunk 1]
```diff
@@ -18,6 +18,16 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
 
 :::
 ## May 2024
+### Version 1.19.1
+
+#### New features and enhancements
+
+- Asset Governance for AWS : Added new governance recommendations and cost computation support for AWS resources including `cache-cluster`, `S3`,`` Redshift`, and `Redshift-snapshot`. [CCM-17852]
```
Comment (brian-f-harness) [inline release-notes/cloud-cost-management.md:25]:
```suggestion
- Asset Governance for AWS : Added new governance recommendations and cost computation support for AWS resources including `cache-cluster`, `S3`, `Redshift`, and `Redshift-snapshot`. [CCM-17852]
```

File: release-notes/cloud-cost-management.md [hunk 2]
```diff
@@ -18,6 +18,16 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+- Editing Enforcements: Improved the enforcement editing process. Now, when viewing and editing an enforcement, it will no longer get enabled automatically if it was previously disabled. This ensures that the enforcement status remains consistent unless intentionally changed by the user. [CCM-18050]
+- Email Validation Enhancement: We've introduced better validation for email addresses in the recipients list for perspectives report. This enhancement ensures that only correctly formatted email addresses are accepted in the recipients list, enhancing data integrity and security. [CCM-17850]
```
Comment (brian-f-harness) [inline release-notes/cloud-cost-management.md:27]:
```suggestion
- Email Validation Enhancement: We've introduced better validation for email addresses in the recipients list for perspectives report. This enhancement ensures that only correctly formatted email addresses are accepted in the recipient's list, enhancing data integrity and security. [CCM-17850, ZD-63324]
```

File: release-notes/cloud-cost-management.md [hunk 3]
```diff
@@ -18,6 +18,16 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+
+#### Fixed issues
+- Azure perspectives Previously, only fields in "group by" with available data were displayed in Azure perspectives, leading to incomplete views and unnecessary errors. With this update, all relevant fields, including those without data, will now be visible. This ensures a comprehensive perspective of the data, eliminating unnecessary errors arising from missing data at the source level. [CCM-17573]
```
Comment (brian-f-harness) [inline release-notes/cloud-cost-management.md:30]:
```suggestion
- Azure perspectives Previously, only fields in "group by" with available data were displayed in Azure perspectives, leading to incomplete views and unnecessary errors. With this update, all relevant fields, including those without data, will be visible. This ensures a comprehensive data perspective, eliminating unnecessary errors from missing data at the source level. [CCM-17573, ZD-62691]
```

File: release-notes/cloud-cost-management.md (hunk 2 above)
Comment (joeyouss) [reply release-notes/cloud-cost-management.md:27]: hey @brian-f-harness the list is actually the recipients list

**Review by brian-f-harness — DISMISSED** [review]

File: release-notes/cloud-cost-management.md [hunk 4]
```diff
@@ -23,10 +23,18 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
 - Editing Enforcements: Improved the enforcement editing process. Now, when viewing and editing an enforcement, it will no longer get enabled automatically if it was previously disabled. This ensures that the enforcement status remains consistent unless intentionally changed by the user. [CCM-18050]
 - Email Validation Enhancement: We've introduced better validation for email addresses in the recipients list for perspectives report. This enhancement ensures that only correctly formatted email addresses are accepted in the recipients list, enhancing data integrity and security. [CCM-17850, ZD-63324]
 - Azure Governance Subscription selection: In the Azure Rule window's Subscription drop-down menu, previously, only the Subscription ID was displayed. Now, both the Subscription Name and ID are shown for better clarity and ease of selection. [CCM-17650]
+- Node New K8s Labelling: We have added support for the new K8s labels starting from K8s v1.17 for Instance Type, Operating System, Region and Zone respectively. We use these labels to get the public pricing data for a given cloud provider. [CCM-17979]
```
Comment (brian-f-harness) [inline release-notes/cloud-cost-management.md:30]:
```suggestion
- Node New K8s Labelling: We have added support for the new K8s labels starting from K8s v1.17 for Instance Type, Operating System, Region, and Zone respectively. We use these labels to get the public pricing data for a given cloud provider. [CCM-17979]
```

File: release-notes/cloud-cost-management.md [hunk 5]
```diff
@@ -23,10 +23,18 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
 #### New features and enhancements
 
 - Asset Governance for AWS : Added new governance recommendations and cost computation support for AWS resources including `cache-cluster`, `S3`,`Redshift`, and `Redshift-snapshot`. [CCM-17852]
-- Asset Governance for GCP : Added new governance recommendations and cost computation support for GCP resources including `redis`, `gke-cluster`,`` bq-dataset`, `function`, `bucket`, `dataflow-job`, `loadbalancer-address`. [CCM-17852]
+- Asset Governance for GCP : Added new governance recommendations and cost computation support for GCP resources including `redis`, `gke-cluster`,` bq-dataset`, `function`, `bucket`, `dataflow-job`, `loadbalancer-address`, `cloud-run-service` and also, added cost computation for `gcp.loadbalancer-forwarding-rule`. [CCM-17852]
```
Comment (brian-f-harness) [inline release-notes/cloud-cost-management.md:26]:
```suggestion
- Asset Governance for GCP : Added new governance recommendations and cost computation support for GCP resources including `redis`, `gke-cluster`, `bq-dataset`, `function`, `bucket`, `dataflow-job`, `loadbalancer-address`, `cloud-run-service` and also, added cost computation for `gcp.loadbalancer-forwarding-rule`. [CCM-17852]
```

**Review by brian-f-harness — APPROVED** [review]

**Comment by joeyouss** [comment]
@brian-f-harness should I go ahead and merge?

File: release-notes/cloud-cost-management.md [hunk 6]
```diff
@@ -18,6 +18,27 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+
+| Old Label | New Label |
+|------------|--------------|
+| beta.kubernetes.io/instance-type | node.kubernetes.io/instance-type |
+| beta.kubernetes.io/os | kubernetes.io/os |
+| failure-domain.beta.kubernetes.io/region | topology.kubernetes.io/region |
+| failure-domain.beta.kubernetes.io/zone | topology.kubernetes.io/zone |
+
+
+#### Fixed issues
```
Comment (SmritiSatya) [inline release-notes/cloud-cost-management.md:40]: Fixed an issue in Azure perspectives which displayed only the fields in the "group by" filter with the available data. This led to errors. Now, all the relevant fields, including those without data, will be visible. This eliminates unnecessary errors arising from missing data at the source level. [CCM-17573, ZD-62691]

**Review by SmritiSatya — APPROVED** [review]

**Pull request merged**